extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/ControllerImpl.java

```diff
@@ -529,6 +532,31 @@ private MSQTaskReportPayload runInternal(final QueryListener queryListener, fina
         countersSnapshot,
         null
     );
+    // Emit summary metrics
+    emitSummaryMetrics(msqTaskReportPayload, querySpec);
+    return msqTaskReportPayload;
+  }
+
+  private void emitSummaryMetrics(final MSQTaskReportPayload msqTaskReportPayload, final MSQSpec querySpec)
+  {
+    long totalProcessedBytes = msqTaskReportPayload.getCounters() != null
+        ? msqTaskReportPayload.getCounters().copyMap().values().stream().mapToLong(
+            integerCounterSnapshotsMap -> integerCounterSnapshotsMap.values().stream()
+                .mapToLong(counterSnapshots -> {
+                  Map<String, QueryCounterSnapshot> workerCounters = counterSnapshots.getMap();
+                  return workerCounters.entrySet().stream().mapToLong(
+                      channel -> {
+                        if (channel.getKey().startsWith("input")) {
+                          ChannelCounters.Snapshot snapshot = (ChannelCounters.Snapshot) channel.getValue();
+                          return snapshot.getBytes() == null ? 0L : Arrays.stream(snapshot.getBytes()).sum();
+                        }
+                        return 0L;
+                      }).sum();
+                }).sum()).sum()
+        : 0;
+
+    log.info("Total processed bytes: %d, query: %s", totalProcessedBytes, querySpec.getQuery());
```

Comment:
This can be a debug log.

```suggestion
    log.debug("Processed bytes[%d] for query[%s].", totalProcessedBytes, querySpec.getQuery());
```